### ISSUE TITLE
updb Fails From Issue With Symphony Console

### ISSUE THREAD

**Issue by peoriadrupaluser** [drush-ops/drush#5654]
**Describe the bug**
After downloading Drupal Core 9.5.9 with Composer, Drush 11.6.0 fails with two fatal errors. There errors are:

PHP Fatal error:  Declaration of Symfony\Component\Console\Style\OutputStyle::getVerbosity() must be compatible with Symfony\Component\Console\Output\OutputInterface::getVerbosity(): int in /var/www/user/vendor/symfony/console/Style/OutputStyle.php on line 78

Fatal error: Declaration of Symfony\Component\Console\Style\OutputStyle::getVerbosity() must be compatible with Symfony\Component\Console\Output\OutputInterface::getVerbosity(): int in /var/www/user/vendor/symfony/console/Style/OutputStyle.php on line 78

**To Reproduce**
This happens when running `drush updb`.

**Expected behavior**
The expected behavior is that the Drush updb command would run completely.  

**Actual behavior**
The updb command starts to run because this output is seen "CiviCRM: System: Please specify a realistic site URL (Ex: drush -l
http://example.com:456 ...)." This output is expected with my install.  After this output the two fatal errors above are printed.  After the fatal errors are shown a "Drush command terminated abnormally."  is shown and the command stops.  

**Workaround**
I have not found a workaround for this.  

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  |   11.6.0
| Drupal version? | 9.5.9
| PHP version     |  8.0.28
| OS?             | Linux

**Additional information**
I do not think it changes anything but I am also using CiviCRM 5.62.0.  CiviCRM is also installed with Composer.

**Comment by greg-1-anderson** [comment]
By "Composer Drush" do you mean `composer global require`, or `composer require` in a directory other than your Drupal site? Either of those configurations will result in the reported symptoms, or something similar. To use Drush 11 or later, run `composer require` from your Drupal site's project root, to add Drush to the same vendor that Drupal is stored in. This will prevent having two conflicting versions of OutputInterface et. al.

**Comment by peoriadrupaluser** [comment]
@greg-1-anderson, I added a comma and it should make more sense now. Sorry about that. 

It now reads "After downloading Drupal Core 9.5.9 with Composer, Drush 11.6.0 fails with two fatal errors. "

Said another way.  I downloaded Drupal Core 9.5.9 with Composer. After completing this download Drush 11.6.0 fails with two fatal errors.

Drush has been installed with Composer globally.  

Thanks!
